### DISCUSSION TITLE
Using point blank and chaos

### DISCUSSION THREAD

**Discussion by Darcii** [3kh0/ext-remover#422] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Hello im not to techy... but i was wondering if you could use the point blank exploit to run a inspect element clone on this system page then continue with the chaos exploit. It would be interesting if it would work.

https://xlak.github.io/chaos/
https://tinyurl.com/chrome-nspect
javascript:javascript:(function()%7B(function() %7Bvar x %3D document.createElement("script")%3Bx.src %3D "https%3A%2F%2Fcdn.jsdelivr.net%2Fgh%2FSnowLord7%2Fdevconsole%40master%2Fmain.js"%3Bx.onload %3D alert("Loaded Developer Console!")%3Bdocument.head.appendChild(x)%3B%7D)()%7D)() 
thats the inspect clone i was talking abt

**Comment by obkduishinijf** [comment]
idk probably

**Comment by Aka-but-nice** [comment, marked as answer]
> Hello im not to techy... but i was wondering if you could use the point blank exploit to run a inspect element clone on this system page then continue with the chaos exploit. It would be interesting if it would work.
> 
> https://xlak.github.io/chaos/ https://tinyurl.com/chrome-nspect javascript:javascript:(function()%7B(function() %7Bvar x %3D document.createElement("script")%3Bx.src %3D "https%3A%2F%2Fcdn.jsdelivr.net%2Fgh%2FSnowLord7%2Fdevconsole%40master%2Fmain.js"%3Bx.onload %3D alert("Loaded Developer Console!")%3Bdocument.head.appendChild(x)%3B%7D)()%7D)() thats the inspect clone i was talking abt

 No, you can't run javascript on chrome pages, ofc if you can find uses of window.open() or canceling a page load, which to my knowledge, there is none in chrome://inspect.
